source/codegen/templates/service_helpers.mako

```diff
@@ -187,7 +187,7 @@ one_of_case_prefix = f'{namespace_prefix}{function_name}Request::{PascalFieldNam
 % elif c_type == 'ViSession':
       auto ${parameter_name}_grpc_session = ${request_snippet};
       ${c_type} ${parameter_name} = session_repository_->access_session(${parameter_name}_grpc_session.id(), ${parameter_name}_grpc_session.name());\
-% elif c_type == 'ViInt32[]':
+% elif c_type == 'ViInt32[]' or c_type == 'ViAddr[]':
       auto ${parameter_name} = const_cast<${c_type_pointer}>(reinterpret_cast<const ${c_type_pointer}>(${request_snippet}.data()));\
```

Comment:
I didn't notice this before. There's no point in doing a `const_cast` of a `reinterpret_cast`. We should just make it a `reinterpret_cast` without the `const`.

Reply:
@jonlugoatx ,
>  We should just make it a reinterpret_cast without the const.

We initially tried this way. But we got the following error saying that "reinterpret_cast cannot cast away const or other type qualifiers".

![Comment_const](https://user-images.githubusercontent.com/77618357/114977693-b8135800-9ea5-11eb-92f7-c5b5919ede16.png)

Then @ganeshnie suggested using const_cast before reinterpret_cast.

Reply:
@jonlugoatx 
I discussed with @ganeshnie and we decided to resolve this comment and submit PR for now. Later if we need any changes, we will do it as part of separate PR.